compiler/src/iree/compiler/Dialect/Flow/Transforms/RaiseSpecialOps.cpp

```diff
@@ -286,18 +305,24 @@ struct RaiseSpecialOpsPass : public RaiseSpecialOpsBase<RaiseSpecialOpsPass> {
     IRRewriter rewriter(&getContext());
 
     getOperation()->walk([&](linalg::GenericOp op) {
-      rewriter.setInsertionPoint(op);
+      linalg::GenericOp linalgOp = op;
 
-      // Try raising to tensor.export.
+      // Try raising to tensor.export and create an intermediate linalg.generic.
+      rewriter.setInsertionPoint(op);
```

Comment:
Typically good to guard insertion point changes with 
```
OpBuilder::InsertionGuard g(rewriter);
```

otherwise the insertion point doesnt get reset and then leads to weird errors.